cp/main.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <fcntl.h>
+#include <unistd.h>
+#include <stdlib.h>
+
+void quick(int er) {
```

Comment:
Suppose program exits with code 17. What will you understand from that ?
Error handling is for you. Log errors in more informative way.
Use "perror".